pkg/controller/controller_instance_test.go

```diff
@@ -344,8 +344,8 @@ func TestReconcileServiceInstanceNonExistentClusterServicePlanK8SName(t *testing
 	events := getRecordedEvents(testController)
 
 	expectedEvent := warningEventBuilder(errorNonexistentClusterServicePlanReason).msgf(
-		"References a non-existent ClusterServicePlan with K8S name %q on ClusterServiceClass with K8S name %q",
-		"nothereplan", testClusterServiceClassGUID,
+		"References a non-existent ClusterServicePlan {ClassName:%q, PlanName:%q}",
```

Comment:
Since you have already tested the formatting of the output using Format method, I don't see a need to reformat it in the same way in the tests. I think you can just use the nice Format method output in the tests too.